source/atomics.tex

```diff
@@ -798,7 +798,7 @@
 
 \pnum
 The implementation shall provide the function template specializations
```

Comment:
@jensmaurer As far as I can tell the atomic_fetch_KEY and atomic_fetch_KEY_explicit nonmembers need to be explicit specializations. Although the wording that allows them to be mapped to a base class of atomic<integral> means it might be unobservable, because the user can't name the base class (and so can't take its address).